Makefile

```diff
@@ -1,70 +1,37 @@
-export GO111MODULE=on
```

Comment:
Is there a particular reason to wholesale delete the scripts to initiate unit testing of the gRPC logic right now, via this PR which is focuses on live reloading?

Reply:
At first I didn't intend to do this, but then I got errors while running the scripts (I believe it was something related to the setting of the ENV vars) and since @nics already declared all that was not used anymore I deleted it.

Reply:
we'll move them to server/Makefile